watch/watch.c

```diff
@@ -32,25 +33,24 @@ static char *progname;
 int main(int argc, char *argv[]);
 void usage(void);
 
-struct Flag {
-	int Beep_on_error, No_title;
-}; static struct Flag flag;
-
 int main(int argc, char *argv[]) {
 	progname = argv[0];
 	int option = 0;
-	struct timespec interval = {0};
-	int c = 0, ec = 0, term_x = 0, term_y = 0;
+	int fBeep_on_error = 0,
+	    fNo_title = 0,
+	    c = 0, ec = 0,
+	    term_x = 0;
 	char **commandv;
 	pid_t exec_pid;
+	struct timespec interval = {0};
 	
-	/* 
+	/*
 	 * Default interval of 2 seconds, as other major
 	 * implementations usually do.
 	 */
 	interval.tv_sec = 2;
 
```

Comment:
Seems like I forgot but it may be important to add an `interval.tv_nsec = 0` here.

Reply:
> Seems like I forgot but it may be important to add an `interval.tv_nsec = 0` here.

I was thinking... wasn't it already defined as 0 when we initialized the ``interval`` struct? (See line 45)

Reply:
POSIX doesn't say anything about the "number" contained in this structure. Anyway, I think it would be better to remove the `= {0}` from the `interval` declaration considering its suspicious implementation-defined behaviour.

Reply:
> POSIX doesn't say anything about the "number" contained in this structure. Anyway, I think it would be better to remove the `= {0}` from the `interval` declaration considering its suspicious implementation-defined behaviour.

D'accord.